src/build.js

```diff
@@ -30,7 +34,8 @@ module.exports = function build(markdown) {
   return `
 ${doImports}
 
-module.exports = function() {
+export const attributes = ${JSON.stringify(camelize(frontMatterAttributes))};
+export default function() {
   return (
     <div>
```

Comment:
I assumed that this would be fine considering that the es6 module import is being used above. Is it not safe to assume that if the`import` syntax works that the `export` syntax will work aswell? I did have a concern about change from `module.exports` to `export default` for the main component because if you are using commonjs to load the module, the require statement will have to change from

```javascript
var Component = require('./my-markdown-file.md');
```

to

```javascript
var Component = require('./my-markdown-file.md').default;
```